.evergreen/.evg.yml

```diff
@@ -960,6 +984,22 @@ tasks:
               export GCPOIDC_TEST_CMD="OIDC_ENV=gcp ./.evergreen/run-mongodb-oidc-test.sh"
               bash $DRIVERS_TOOLS/.evergreen/auth_oidc/gcp/run-driver-test.sh
 
+    - name: "oidc-auth-test-k8s"
+      commands:
+        - command: ec2.assume_role
+          params:
+            role_arn: ${aws_test_secrets_role}
+            duration_seconds: 1800
+        - func: "oidc-auth-test-k8s-func"
+          vars:
+            VARIANT: eks
+#        - func: "oidc-auth-test-k8s-func" TODO disabled, memory issue, consider forking and increasing here: https://github.com/mongodb-labs/drivers-evergreen-tools/commit/4bc3e500b6f0e8ab01f052c4a1bfb782d6a29b4e
+#          vars:
+#            VARIANT: gke
```

Comment:
Variant is disabled due to [failure](https://evergreen.mongodb.com/task_log_raw/mongo_java_driver_testk8soidc_variant_oidc_auth_test_k8s_patch_241ef1cf9889c94a77558b16c46abeb21686cb16_6752086807a71d0007a2985d_24_12_05_20_09_12/0?type=T):

```
 [2024/12/05 13:34:17.109] Compiling build file '/tmp/test/build.gradle' using BuildScriptTransformer.
 [2024/12/05 13:34:17.109] Starting process 'command 'git''. Working directory: /tmp/test Command: git describe --tags --always --dirty
 [2024/12/05 13:34:22.782] debconf: delaying package configuration, since apt-utils is not installed
 [2024/12/05 13:34:22.782] command terminated with exit code 137
```

See note in TODO comment (must be removed before merging).